src/argilla/server/security/settings.py

```diff
@@ -0,0 +1,67 @@
+#  coding=utf-8
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import os
+from typing import TYPE_CHECKING, Optional
+from uuid import uuid4
+
+from argilla.server.pydantic_v1 import BaseSettings, validator
+
+
+class Settings(BaseSettings):
+
+    """
+    Attributes
+    ----------
+
+    secret_key:
+        The secret key used for signed the token data
+
+    algorithm:
+        Encryption algorithm for token data
+
+    token_expiration_in_minutes:
+        The session token expiration in minutes. Default=30000
+
+    """
+
+    secret_key: str = uuid4().hex
+    algorithm: str = "HS256"
+
+    token_expiration_in_minutes: int = 15
+    token_expiration: Optional[int]
```

Comment:
I think it's a little bit confusing to have a `token_expiration_in_minutes` and at the same time `token_expiration`. I believe we should go for only `token_expiration` (using seconds) to avoid unnecessary complex scenarios (for example specifying values to the two settings at the same time).